main/data/conversation_details.ui

```diff
@@ -122,7 +122,8 @@
                           </object>
                         </child>
                         <child>
-                          <object class="GtkButton" id="block_button">
+                          <object class="AdwSplitButton" id="block_button">
+                            <property name="menu_model">block_menu_model</property>
                             <property name="visible" bind-source="model" bind-property="show-blocked" bind-flags="sync-create"/>
                             <child>
                               <object class="AdwButtonContent" id="block_button_content">
```

Comment:
@fiaxh , like in 384ef1d3f15c82af1a4647b0ee704f77919ff2b1, should `action-unavailable-symbolic` also be replaced by `dino-block-symbolic`  here? thanks

Reply:
Yes, please!